### PR TITLE
implement new deployment style under temporary deploy command

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

File: src/root-command.ts [hunk 1]
```diff
@@ -25,19 +28,38 @@ export default abstract class extends Command {
...
-        : 'localhost'
-      this._mesg = application({
-        endpoint: `${host}:50052`
-      })
+      this._mesg = application({endpoint: this.engineEndpoint})
     }
     return this._mesg
   }
 
+  get serviceSDK() {
```
Comment (antho1404) [inline src/root-command.ts]: would rename with API instead of SDK but not blocking

**Comment by NicolasMahe** [comment]
Can this CLI only use the mesg-js lib to access the gRPC apis?
The mesg-js should be updated to show ALL gRPC apis.

**Comment by ilgooz** [comment]
> Can this CLI only use the mesg-js lib to access the gRPC apis?
> The mesg-js should be updated to show ALL gRPC apis.

yes but we should do it when we have a clear vision about how to structure our new gRPC apis in the mesg-js lib

File: src/root-command.ts (hunk 1 above)
Comment (ilgooz) [reply src/root-command.ts]: done

**Comment by antho1404** [comment]
I'm totally ok to have this directly in the CLI the time we actually stabilize all this and then extract it in the mesg-js app if needed.

**Comment by antho1404** [comment]
If you want we can hide this command with `static hidden = true` https://oclif.io/docs/commands

**Comment by NicolasMahe** [comment]
I got a weird error when I want to start a service deployed with this new command:
```
➜  cli git:(feature/new-deploy) ./bin/run service:start CfXKjFHwNo8GXyyg5qpEWaHzb6ec27NCma52k1hgUEBw
Start service... !
Error: 2 UNKNOWN: Error response from daemon: rpc error: code = InvalidArgument desc = name must be valid as a DNS name component
    at Object.exports.createStatusError (~/Developments/MESG/cli/node_modules/grpc/src/common.js:91:15)
    at Object.onReceiveStatus (~/Developments/MESG/cli/node_modules/grpc/src/client_interceptors.js:1204:28)
    at InterceptingListener._callNext (~/Developments/MESG/cli/node_modules/grpc/src/client_interceptors.js:568:42)
    at InterceptingListener.onReceiveStatus (~/Developments/MESG/cli/node_modules/grpc/src/client_interceptors.js:618:8)
    at callback (~/Developments/MESG/cli/node_modules/grpc/src/client_interceptors.js:845:24)
```

I previously executed the command:
```
./bin/run service:newdeploy "$(./bin/run service:compile ../services/service-ethereum)"
```
It returned:
```
Deploy service... com.mesg.ethereum (CfXKjFHwNo8GXyyg5qpEWaHzb6ec27NCma52k1hgUEBw)
```
The command service list of the Go cli show this service:
```
➜  ~ mesg-core service list
HASH                                            SID                  NAME               STATUS     
CfXKjFHwNo8GXyyg5qpEWaHzb6ec27NCma52k1hgUEBw    com.mesg.ethereum    Ethereum           stopped    
3CaMQXGGRZbhrTX1uk3yTwcyBEja4nHbNuxSuuW2tug6    marketplace          Marketplace        running    
DUKu7QydSpbSQXxch3ZNrJYWXMigh3iwC9EnNgQeRmjw    ethwallet            Ethereum Wallet    running  
```

**Pull request merged**